.github/policy.yml

```diff
@@ -0,0 +1,34 @@
+policy:
+  approval:
+    - or:
+      - and:
+          - tests passed
+          - es has approved
+      - and:
+          - dependabot is making the PR
+          - or:
+              - tests passed
+              - es has approved
+  disapproval:
+    requires:
+      organizations:
+        - "bestseller"
+approval_rules:
+  - name: tests passed
+    if:
+      has_successful_status:
+        - "GitGuardian Security Checks"
```

Comment:
Shouldn't we add the "Go Test" here, so it won't merge until it has completed with success?

Reply:
Yes